UI/MapWnd.h

```diff
@@ -65,19 +65,22 @@ class MapWnd : public GG::Wnd {
       * save-and-load cycle */
     void                        GetSaveGameUIData(SaveGameUIData& data) const;
 
-    /** returns true iff this MapWnd is visible and usable for interaction, but
-      * the allowed interactions are restricted to those appropriate to the
-      * production screen */
+    /** returns true if MapWnd is visible and usable behind a production window.
+     * MapWnd interactions are restricted to those appropriate to the production window */
     bool                        InProductionViewMode() const;
 
-    /** returns true iff this MapWnd is visible and usable for interaction, but
-      * the allowed interactions are restricted to those appropriate to the
-      * research screen */
+    /** returns true if MapWnd is visible and usable behind a research window.
+     * MapWnd interactions are restricted to those appropriate to the research window.
```

Comment:
It's longer now, but I still don't know why the 2nd and later lines of this comment are here... What does this discussion have to do with this function?